docker/docker_client.go

```diff
@@ -78,7 +78,7 @@ func newDockerClient(refHostname, certPath string, tlsVerify bool) (*dockerClien
 	}, nil
 }
 
-func (c *dockerClient) makeRequest(method, url string, headers map[string]string, stream io.Reader) (*http.Response, error) {
+func (c *dockerClient) makeRequest(method, url string, headers map[string][]string, stream io.Reader) (*http.Response, error) {
```

Comment:
Would this work, and be cleaner, to use the existing `http.Header` typedef? Especially looking at `docker_image_dest` having to copy this type declaration.